docker/lr-metrics/Dockerfile

```diff
@@ -21,7 +22,7 @@ RUN conda env create -f /environment.yml && conda clean -a
 RUN git clone https://github.com/broadinstitute/gatk.git -b kvg_pbeap \
     && cd gatk \
     && ./gradlew \
-    && cp /gatk/build/libs/gatk-package-4.1.0.0-54-g47086b6-SNAPSHOT-local.jar /usr/local/bin/gatk.jar \
+    && cp /gatk/build/libs/gatk-package-4.1.0.0-57-gc949722-SNAPSHOT-local.jar /usr/local/bin/gatk.jar \
```

Comment:
can you briefly explain the changes between these two snapshots?

Reply:
There was a bug in the metrics for PacBio machines - the ZMW values can now be larger than maxint and it was throwing an error.  I updated this branch to cast to `long` so that we wouldn't have the issue.

Reply:
Add a line to checkout the revision that we're using now (before line 24).